app/src/test/java/helper/UtilsTest.kt

```diff
@@ -0,0 +1,19 @@
+package helper
+
+import junit.framework.Assert.assertEquals
+import org.fossasia.susi.ai.helper.Utils
+import org.junit.Test
+
+/**
+ *
+ * Created by arundhati24 on 03/08/2018.
+ */
+class UtilsTest {
+    @Test
+    fun testTruncateEmailAtEnd() {
+        assertEquals(true, Utils.truncateEmailAtEnd("testuser@example.com").equals("testuser@ ..."))
```

Comment:
Wrong use of assetEquals

Reply:
@iamareebjamal assertEquals() takes two params expected value and actual value, so is it because of the passed params?

Reply:
`truncateEmailAtEnd` returns String and not boolean. So expected param should be?